news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+*written by [Nepuri](https://osu.ppy.sh/users/6637817)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/hibi.jpg)](https://osu.ppy.sh/beatmapsets/1006568)
+
+Crazy snappings? Stupidly fast sections? A ton of finishers? SV to strain your eyes? This map has it all. In 4th place we have KikuoHana - Hibi Darake no Me by [Alchyr](https://osu.ppy.sh/users/4993032). 
+
+Not many people can equal a crazy song with a crazy map, but this guy is one of the best at it. With three difficulties to cater for more experienced to the most experienced players, it's not hard to tell the mapper has put in a lot of work into this mapset and to make it a nightmare to get a full combo. 
+
+The map starts off with the first half being deceivingly calm, until the drop hits and the map explodes into a burst of fun. All difficulties (with the top diff in particular) utilise a wide range of tough patterns constantly varying in length, rhythm and snappings which prove to be difficult for even the best of players, and it's one and a half minutes of hell which shows how creative mappers, especially Alchyr, are getting with their work and how high the bar for map difficulty has been set.
+
+*written by [\_DUSK\_](https://osu.ppy.sh/users/6092181)*
+
+## <a id="catch"></a> osu!catch
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/chocolate.jpg)](https://osu.ppy.sh/beatmapsets/1061800)
+
+Spring is coming, wanna play a map with spring flavour? 'hot chocolate' by [JBHyperion](https://osu.ppy.sh/users/4879508) should be one of the best choices here. Needless to say, the high-quality mapping style is always a mark for featured artist maps, this map is no exception. 
+
+Despite the extraordinary clean and fresh flows, the mapper also tried some new features here - extending sliders and 1/8 Hyperdashes. These innovative patterns bring the map to the next level and make the map stands out as well. Please make a cup of hot chocolate and enjoy this fantastic map!
+
+*written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/loving.jpg)](https://osu.ppy.sh/beatmapsets/1080025)
+
+'How on earth did eroge get into spotlights?' may be a question that pops up when first encountering this set. Despite the general stigma against the genre, [Luminate](https://osu.ppy.sh/users/4778689)'s "Loving Trip" does a great job of following the song. 
+
+Slower parts of the song keep things interesting with varied rhythm and patterns, while higher density sections have liberal use of antiflow that contrasts well. This is one of the few sets that I've genuinely had fun playing this year, and I hope you give it a shot as well.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
```

Comment:
100% missed pun opportunity for 'visually novel'

Reply:
not interested to apply this